Sources/SwiftKueryORM/Model.swift

```diff
@@ -616,7 +621,274 @@ public extension Model {
             } catch {
               onCompletion(nil, RequestError(.ormIdentifierError, reason: "Could not construct Identifier"))
             }
+          }
+          onCompletion(result, nil)
+        }
+      }
+    }
+  }
+
+  /// Find all the models matching the QueryParams
+  /// - Parameter using: Optional Database to use
+  /// - Parameter matching: Optional QueryParams to use
+  /// - Returns: An array of model
+  static func findAll<Q: QueryParams>(using db: Database? = nil, matching queryParams: Q, _ onCompletion: @escaping ([Self]?, RequestError?) -> Void) {
+    guard let database = db ?? Database.default else {
```

Comment:
"Getting a connection" is so common that maybe it should be factored out into a common function?

Reply:
This can be done in another refactoring PR?